ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx

```diff
@@ -60,7 +60,7 @@ export const getServerSideProps: GetServerSideProps = async context => {
           }
         : null,
       pageType: UGC_PAGE,
-      pathname: null,
+      pathname: context.resolvedUrl,
       service,
```

Comment:
There are lots of things in these props that we're not setting, when we are setting them for the live pages. Is this intentional? Do we need a ticket written to fill these in ?

Reply:
If its simple to add now that would be great. But if its a lot more work, a dedicated ticket would be better.

Reply:
okay, ill make a seperate ticket for it as i dont think it falls under this 👍

Reply:
actually thinking about it i think its simple enough to add as part of this, sorry hadnt refreshed and seen your comment !

Reply:
Awesome, thanks for this!